### ISSUE TITLE
Build script doesn't work with Dash shell

### ISSUE THREAD

**Issue by sammoth** [fabioarnold/TwoTriangles#8]
The build instructions and PKGBUILD fail if Dash is the default shell - should it call Bash instead?

**Comment by fabioarnold** [comment]
Well, the shebang specifies /bin/bash. But I'm open to suggestions on how to make the build script compatible.

**Comment by sammoth** [comment]
Yep, but near the bottom of the readme it says to run `sh build.sh run`, and also in the PKGBUILD it calls `sh` https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=twotris-git
